helper.go

```diff
@@ -277,6 +271,114 @@ func arrayToKV(m map[string]RedisMessage, arr []RedisMessage, keys []string) map
 	return m
 }
 
+func clusterMGet(client Client, ctx context.Context, keys []string) (ret map[string]RedisMessage, err error) {
+	ret = make(map[string]RedisMessage, len(keys))
+	if len(keys) == 0 {
+		return ret, nil
+	}
+
+	slotIdx := make(map[uint16]int, len(keys)/2)
+	var builders []any
```

Comment:
```suggestion
	cmds := mgetcmdsp.Get(0, len(keys))
	defer mgetcmdsp.Put(cmds)
```

Again, we should avoid additional allocations. This `builders []any` can be replaced by the `cmds.s`